R/epiprocess-data.R

```diff
@@ -66,87 +74,117 @@
-#' \href{https://creativecommons.org/licenses/by/4.0/}{Creative Commons Attribution 4.0 International license}
-#' by the Johns Hopkins University on behalf of its Center for Systems Science in Engineering.
-#' Copyright Johns Hopkins University 2020.
+#' @source This object contains a modified part of the
+#'   \href{https://github.com/CSSEGISandData/COVID-19}{COVID-19 Data Repository by the Center for Systems Science and Engineering (CSSE) at Johns Hopkins University} as
+#'   \href{https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/jhu-csse.html}{republished in the COVIDcast Epidata API}.
+#'   This data set is licensed under the terms of the
+#'   \href{https://creativecommons.org/licenses/by/4.0/}{Creative Commons Attribution 4.0 International license}
+#'   by the Johns Hopkins University on behalf of its Center for Systems
+#'   Science in Engineering. Copyright Johns Hopkins University 2020.
 #'
 #' Modifications:
 #' * \href{https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/jhu-csse.html}{From the COVIDcast Epidata API}:
-#' These signals are taken directly from the JHU CSSE \href{https://github.com/CSSEGISandData/COVID-19}{COVID-19 GitHub repository} without changes.
-#' * Furthermore, the data has been limited to a very small number of rows, the signal names slightly altered, and formatted into a tibble.
-"covid_incidence_outliers"
+#'   These signals are taken directly from the JHU CSSE
+#'   \href{https://github.com/CSSEGISandData/COVID-19}{COVID-19 GitHub repository}
+#'   without changes. The 7-day average signals are computed by Delphi by
+#'   as moving averages of the preceding 7 days, so the signal for
+#'   June 7 is the average of the underlying data for June 1 through 7,
+#'   inclusive.
+#' * Furthermore, the data has been limited to a very small number of rows,
+#'   formatted into a tibble, and the signal names slightly altered.
+"covid_incidence_county_subset"
 
-#' Subset of JHU daily cases from counties in Massachusetts and Vermont
-#'
-#' This data source of confirmed COVID-19 cases and deaths
-#' is based on reports made available by the Center for
-#' Systems Science and Engineering at Johns Hopkins University.
-#' This example data ranges from Mar 1, 2020 to Dec 31, 2021,
-#' and is limited to Massachusetts and Vermont.
-#'
-#' @format A tibble with 16,212 rows and 5 variables:
+# Epiprocess Vignette Data ----------------------------------------------------
+
+#' Subset of JHU daily COVID-19 cases from New Jersey and Florida
+#' @description
+#' This data source of confirmed COVID-19 cases is based on reports made
+#' available by the Center for Systems Science and Engineering at Johns
+#' Hopkins University. This example data is downloaded from the CMU Delphi
+#' COVIDcast Epidata API. It is a snapshot as of Oct 28, 2021 and captures the
```

Comment:
issue?: the `as_of` attached in the metadata is 2023-06-07... which one is correct?